src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -130,6 +136,31 @@ class EmojiPickerMenu extends Component {
         this.setState({selection: event.nativeEvent.selection});
     }
 
+    /**
+     * Calculate the initial filtered + header emojis and header row indices
```

Comment:
```suggestion
     * Calculate the filtered + header emojis and header row indices
```

It looks like this function is used any time the component updates, not just initially.